### ISSUE TITLE
`Checkbox` Border Color When `is_on = False` Should Default to Secondary Color

### ISSUE THREAD

**Issue by Sn3llius** [rio-labs/rio#93]
When `is_on = False` for the `Checkbox` component, the border color defaults to black. For better visual consistency, the border color should match the color of the filled checkbox, such as the secondary color by default.

<img width="122" alt="image" src="https://github.com/rio-labs/rio/assets/41641225/cc520d22-94c2-4139-bc69-f0dc48afb031">

<img width="67" alt="image" src="https://github.com/rio-labs/rio/assets/41641225/cf46f297-d37d-45c8-b7d9-ee6373d9754b">

**Comment by mad-moo** [comment]
I disagree with this. Having checkboxes light up even when disabled seems like way too much visual clutter. It's also not in line with the reference material design checkboxes.

![image](https://github.com/rio-labs/rio/assets/24632992/f0b9a7e4-1afb-45a9-b63f-5dddaaaaaf06)

Finally, I'd argue this _lowers_ consistency, as switches also turn gray when disabled

**Comment by Nirajan1-droid** [comment]
am i suppose to work on it?
@Sn3llius

**Comment by Sn3llius** [comment]
@Nirajan1-droid I discussed this internally with mad-moo, and he has already fixed the issue, as demonstrated in his picture.

**Comment by Nirajan1-droid** [comment]
Thanks for the update.
will keep an eye out for issues to contribute in the future.

**Comment by Nirajan1-droid** [comment]
could you please mention me after opening feature request ?
